runtime/compiler/optimizer/HotFieldMarking.cpp

```diff
@@ -35,24 +35,28 @@
 
 #include <map>
 
+/**
+ * A structure to represent the statistics related to each field during a compilation
+ */
 struct SymStats
    {
-   int32_t _count;
-   int32_t _score;
+   int32_t _count; /* number of blocks within the compilation that contributes to the frequency score value of the field */
```

Comment:
please make these doxygen comments rather than just inline comments so they can come out in the documentation.